backends/tfhe-cuda-backend/cuda/src/crypto/gadget.cuh

```diff
@@ -6,10 +6,11 @@
 
 /**
  * GadgetMatrix implements the iterator design pattern to decompose a set of
- * num_poly consecutive polynomials with degree params::degree. A total of
- * level_count levels is expected and each call to decompose_and_compress_next()
- * writes to the result the next level. It is also possible to advance an
- * arbitrary amount of levels by using decompose_and_compress_level().
+ * num_poly consecutive polynomials with degree compression_params::degree. A
+ * total of level_count levels is expected and each call to
+ * decompose_and_compress_next() writes to the result the next level. It is also
+ * possible to advance an arbitrary amount of levels by using
+ * decompose_and_compress_level().
```

Comment:
Maybe no need to reformat this comment

Reply:
CLion changed it when I refactored a variables name. I will undo it.